templates/cluster-template.yaml

```diff
@@ -50,13 +50,13 @@ spec:
     preKubeadmCommands:
       - swapoff -a
   infrastructureTemplate:
```

Comment:
Does 'infrastructureTemplate' work with clusterctl v1.0.0+?
In my case, I had to replace this to the following 
```
  machineTemplate:
    infrastructureRef:
```